### DISCUSSION TITLE
JupyterLite using Pyodide latest

### DISCUSSION THREAD

**Discussion by lesteve** [pyodide/pyodide#3787] [Show and tell]
Try it at: https://lesteve.github.io/jupyterlite-pyodide-dev/

Repo: https://github.com/lesteve/jupyterlite-pyodide-dev

Pros: this is useful to have a more user-friendly interface than [the Pyodide latest console](https://pyodide.org/en/latest/console.html) when writing non-trivial Python code.

Cons: this is inside a WebWorker so I have no idea how you would do the equivalent of `await pyodide.runPythonAsync` in the browser console ...

**Comment by rth** [comment]
Great, thanks! I also often wanted to test something in JupyterLite as it's certainly a nicer UI than the REPL we have, but not having a easy way to use the dev version was a blocker.

**Comment by jtpio** [comment]
Nice thanks @lesteve for putting that together!
